weather/config.py

```diff
@@ -0,0 +1,62 @@
+"""
+Configuration handling utilities.
+"""
+
+import logging
+import os
+from pathlib import Path
+
+import yaml
+
+logger = logging.getLogger(__name__)
+
+
+def load_config(config_path="settings.yaml"):
+    """
+    Load configuration from YAML file and resolve paths.
+
+    Args:
+        config_path: Path to the YAML configuration file
+
+    Returns:
+        dict: Configuration with resolved paths
+    """
+    if not os.path.exists(config_path):
+        logger.warning(
+            f"Configuration file {config_path} not found. Using default values."
+        )
+        return {
+            "ROOT_DIR": str(Path.cwd()),
+            "input_dir": str(Path.cwd() / "input"),
+            "file_name_base": "westfalia_2024_06_01-2024_06_07",
+            "batch_size": 1000,
+            "log_file": None,
+        }
```

Comment:
I think this does not make much sense. We should raise an error here. If we can't find the configuration we shouldn't just convert the test data in the input folder